### PR TITLE
Suggested updates to the FCM based on work w/ designers and v1 usage

### PR DESCRIPTION
**Pull request by jimmynotjim**
Suggested updates to the FCM based on testing in the cfgov-refresh project and feedback from @huetingj, @duelj, @ajbush, and @schaferjh.

Use https://github.com/virginiacc/cf-layout/pull/2/files?w=0 to ignore whitespace changes

I was unable to run grunt due to the changes to concat less file in the v1.0 release. You'll want to rebase w/ the `cfpb/gh-pages` branch before merging this PR.

### CODE REVIEW

File: src/cf-layout.less [hunk 1]
```diff
@@ -1407,196 +1407,111 @@
...
+      height: 100%;
+      max-width: initial;
+      position: relative;
       left: 50%;
-      transform: translate(-50%,0); 
-      max-width: 540px;
-      width: 540px;
+      transform: translateX(-50%);
     });
   }
```
Comment (virginiacc) [inline src/cf-layout.less:1519]: Do there need to be separate styles here for IE8, since it doesn't support transforms?

**Comment by virginiacc** [comment]
@jimmynotjim  This looks good to me -- I only see one small issue with the image positioning on IE8. (Unless we're moving away from support for IE8?)

File: src/cf-layout.less (hunk 1 above)
Comment (jimmynotjim) [inline src/cf-layout.less:1519]: Good catch, pushing up a commit right now to fix this.

**Comment by jimmynotjim** [comment]
@virginiacc if you have time before Thur, could you take a look at my fix for older IE? Hoping to get this in cfgov-refresh soon.

**Comment by virginiacc** [comment]
@jimmynotjim I'm still seeing issues with the image in IE8 (via Saucelabs):

screen shot 2015-11-25 at 4 21 34 am

vs modern browser:

screen shot 2015-11-25 at 4 27 20 am

**Comment by jimmynotjim** [comment]
@virginiacc sorry, it seems the IE styles task was ignoring the `initial` value and not copying it. Same thing happens with `unset` so we have to use `none`. Apologies for not finding that issue before, but it looks good in Sauce now.

screen shot 2015-11-25 at 11 59 11 am

**Comment by virginiacc** [comment]
Thanks, @jimmynotjim -- looks awesome! :+1: Merged from cl.

**Pull request closed without merging**